.azure/templates/run-qns.yml

```
@@ -0,0 +1,50 @@
+#
+# This pipeline executes a single run of the interop runner between a chosen
+# client and server implementation.
+#
+
+jobs:
+  - ${{ each client in parameters.clients }}:
+    - ${{ each server in parameters.servers }}:
+      - job:
+        displayName: ${{ client }} -> ${{ server }}
+        variables:
+          runCodesignValidationInjection: false
+        steps:
+        - script: |
+            sudo apt install -y cmake libglib2.0-dev libgcrypt20-dev flex bison \
+              byacc libpcap-dev libssh-dev libsystemd-dev ninja-build
+            wget https://www.wireshark.org/download/src/wireshark-latest.tar.xz
+            tar -xvf wireshark-latest.tar.xz
+            rm wireshark-latest.tar.xz
+            mv ./wireshark-* ./wireshark
+            cd wireshark
+            cmake -GNinja DBUILD_wireshark=0 .
```

A `-` is missing. Additionally, with sdjournal disabled, you don't need `libsystemd-dev` and with sshdump/ciscodump disabled, you don't need `libssh-dev` above:
```suggestion
            cmake -GNinja -DBUILD_wireshark=0 -DBUILD_sdjournal=0 -DBUILD_sshdump=0 -DBUILD_ciscodump=0 .
```